profiles/bicycle.lua

```diff
@@ -485,9 +489,21 @@ function way_function (way, result)
   end
 
 
+
   -- maxspeed
   limit( result, maxspeed, maxspeed_forward, maxspeed_backward )
 
+
+  -- not routable if no speed assigned
+  -- this avoid assertions in debug builds
+  if result.forward_speed <= 0 then
```

Comment:
this added check seems kind of wrong to me. I feel we should not by default and only allow if a positive speed is set. How do you feel about this?
It seems that many other locations set the mode in combination with the speed (just from my first impression on searching for `forward_mode`)